test/integration/scenarios/network/peers.js

```diff
@@ -142,6 +142,46 @@ module.exports = function(params) {
 					});
 				});
 			});
+
+			describe('network height after 30 seconds', () => {
```

Comment:
Makes no sense to multiply describe block. There is identical `network height after 30 seconds` block above. 
Please add the data you need (mapped networkHeights) gathered from peer to the `getNetworkStatus` function and use the result in expect block.